PWGGA/GammaConv/macros/AddTask_GammaConvV1_PbPb.C

```diff
@@ -53,8 +53,18 @@ void AddTask_GammaConvV1_PbPb(
   Bool_t    enablePlotVsCentrality        = kFALSE,
   Bool_t    processAODcheckForV0s         = kFALSE,   // flag for AOD check if V0s contained in AliAODs.root and AliAODGammaConversion.root
   // subwagon config
-  TString   additionalTrainConfig         = "0"       // additional counter for trainconfig + special settings
-)  {
+  TString   additionalTrainConfig         = "0",       // additional counter for trainconfig + special settings
+  
+  // ML parameters
+  Bool_t    enablePhotonTree      =  kTRUE,
```

Comment:
The additional train config needs to be the last argument in order to properly set up sub-wagons I think. 
Also, you should set the default values to false

Reply:
Hello @jokonig 
Yes, setting the value to kFalse is good idea. I'll change the order and set the value

Reply:
Hi @an15ms157 ,
Sorry for the delay, I am at a conference this week and missed that.
I still think that putting the values after the additional train config will break the sub-wagon configuration on the lego-train page. Because you can only have one additional argument in the subwagon section (at least to my knowlage). And then, one could not change your new parameters anymore